panda/src/display/CMakeLists.txt

```diff
@@ -90,6 +90,7 @@ composite_sources(p3display P3DISPLAY_SOURCES)
 add_component_library(p3display SYMBOL BUILDING_PANDA_DISPLAY
   ${P3DISPLAY_HEADERS} ${P3DISPLAY_SOURCES})
 target_link_libraries(p3display p3cull p3pgraphnodes)
+target_use_packages(p3display X11)
```

Comment:
Was there something stopping p3display from building on X11 platforms without this? As far as I know, p3display is independent of display API, and shouldn't depend on X11.

Reply:
`display` uses the X11 headers to pull in the `Window` typedef only, because of `NativeWindowHandle::X11Handle`.  Although, it looks like `Window` is just an unsigned long, so I'm not sure why we had a particular need to do that.